mod/wikirate/spec/set/right/wikirate_link_spec.rb

```diff
@@ -8,7 +8,7 @@
   # end
   # it "should block changing the url for existing source" do
   #   # create the page with source
-  #   url = "http://www.google.com/?q=wikirateissocoolandawesomeyouknowsigh"
+  #   url = "http://www.google.com/?q=wikiratesigh"
```

Comment:
that was a global search and replace of `wikirateissocoolandawesomeyouknow` :smile: